doc/demo.rst

```diff
@@ -54,22 +55,32 @@ installation by running the ``hello-world`` container:
    For more examples and ideas, visit:
     https://docs.docker.com/get-started/
 
-Once docker itself is running, you’ll also need to `install the docker
+.. hint::
+   If you get a permission error: "permission denied while trying to connect
+   to the Docker daemon socket", this means you lack the proper permissions to
+   run docker.  The best solution in this case is to have your sysadmin add you
+   to the ``docker`` group.  See `the Docker documenation on running docker as
+   a non-root user
+   <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`__.
+
```

Comment:
> When running “docker compose up --detach alpdb”, got:
`unable to get image 'mysql:latest': permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.48/images/mysql:latest/json": dial unix /var/run/docker.sock: connect: permission denied`
^ Fixed by adding sudo. Should maybe address this in the demo or change commands throughout demo to have sudo?

I've added this hint to address this and point to the relevant Docker docs: better than running sudo is to join the `docker` group, which will give you access to the socket.